internal/logger/logger.go

```diff
@@ -0,0 +1,90 @@
+package logger
+
+import (
+	"go.uber.org/zap"
+	"net/http"
+	"time"
+)
+
+var sugar zap.SugaredLogger
```

Comment:
```suggestion
var Log zap.SugaredLogger
```
Давай сделаем его глобальным и будем использовать во всем коде - `logger.Log.Info....`